test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSRunner.scala

```diff
@@ -181,10 +203,6 @@ final class ScalaJSRunner private[testadapter] (
     val slave = framework.createRunner(slaveLauncher)
     slave.start()
 
-    // Create a runner on the slave
-    slave.send("newRunner")
-    ComUtils.receiveLoop(slave)(msgHandler(slave) orElse ComUtils.doneHandler)
```

Comment:
Why don't we do this when the slave is created? Why do we wait until it's requested?